cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -210,8 +210,18 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodesList() {
     marker.scale.x = kTrajectoryLineStripMarkerScale;
     marker.pose.orientation.w = 1.0;
     for (const auto& node : single_trajectory) {
-      marker.points.push_back(ToGeometryMsgPoint(
-          (node.pose * node.constant_data->tracking_to_pose).translation()));
+      ::geometry_msgs::Point node_point = ToGeometryMsgPoint(
+          (node.pose * node.constant_data->tracking_to_pose).translation());
+      marker.points.push_back(node_point);
+      // Work around the 16384 point limit in rviz by splitting the
+      // trajectory into multiple markers
+      if (marker.points.size() == (1 << 14)) {
```

Comment:
Consistency and readability: Please change `(1 << 14)` to `16384` to match the comment.